src/Footer.jsx

```diff
@@ -0,0 +1,50 @@
+const Footer = ({
+	pageNumber,
+	setPageNumber,
+	numArticles,
+	articlesPerPage,
+}) => {
+	const totalPages = Math.ceil(numArticles / articlesPerPage);
+	const pageNumbers = [<span key="0">Page: </span>];
```

Comment:
Consider a refactor for clarity here - you can ALWAYS render the span - but conditionally render a different amount of buttons